### ISSUE TITLE
USB Not Recognized on Denverton Motheboard

### ISSUE THREAD

**Issue by tlsnine** [pocopico/tinycore-redpill#6]
Sorry for the vague title. 
I have a super micro 1U server with a Denverton/Atom C3758 SOC and I cannot make a USB that’s bootable for it from your img file and it’s making me nuts!!

I’ve tried 5 different USB drives. 6 different OS’s (Mac, Windows and Linux) and a half dozen or so burning methods (Rufus, dd, etcher, win32something, unetbootin, etc), along with toggling MBR and GPT. So, easily 30+ variations of the boot drive and not one is seen by my system. 
I select the boot partition and my server just goes back to the boot selection prompt. 

The same USB drive will boot on an old Intel NUC fine!! Grrr…
What the heck???

The board is capable of booting EFI and legacy. For reference I have used xpenology 6.2 on it from USB as well as ESXi 7, plus I have UBUNTU 20.04 installed on an NVME drive right now. 

TL;DR: Denverton board isn’t recognizing the Tinycore boot drive regardless of which usb drive I use or how it’s configured. 

Any assistance is greatly appreciated!!

I can provide more details. 
thanks!

**Comment by pocopico** [comment]
If you leave the board option to EFI are you able to get into GRUB

**Comment by tlsnine** [comment]
I've set the board for EFI-only, Legacy-only, and autodetect/both.

I've also tried USB2 and USB3 devices, multiple brands and sizes.
FWIW If i download the TinyCore iso from the official site and restored it to USB, it still wont boot on my server.
I have to think it's an issue with TinyCore TBH but while I'm confident using Linux and the terminal, it's a little out of my league at this time.
I'm going to try an older build of TinyCore later just for shnitzngiggles 
Thanks again!

**Comment by tlsnine** [comment]
I updated the bios on my server, set to defaults, and then enabled CSM Support and I was able to boot the USB. 

Please consider this issue closed
